package.json

```diff
@@ -6,13 +6,17 @@
     "@vuepress/plugin-medium-zoom": "^1.8.0",
     "hallmark": "^3.1.0",
     "husky": "^5.0.6",
+    "npm-run-all": "^4.1.5",
+    "svglint": "^1.0.7",
     "vuepress": "^1.8.0",
     "vuepress-plugin-mermaidjs": "^1.8.1"
   },
   "scripts": {
     "dev": "vuepress dev --no-clear-screen .",
     "build": "vuepress build .",
-    "test": "hallmark '**/*.md'",
+    "test": "npm-run-all test:*",
+    "test:md": "hallmark '**/*.md'",
+    "test:svg": "svglint $(git ls-tree -r main --name-only | grep \".svg\")",
     "fix": "hallmark fix '**/*.md'",
```

Comment:
`npm run fix` is what is currently called from the husky pre-commit hook. Wouldn't we want to include the svg linting in this hook, too?

Reply:
We could do, although this linter will not fix anything, just error.